src/handlers/shared/get-user-task-limit-by-role.ts

```diff
@@ -0,0 +1,25 @@
+import { Context } from "../../types";
+
+interface MatchingUserProps {
+  role: string;
+  limit: number;
+}
+
+export async function getUserRole(context: Context, user: string): Promise<MatchingUserProps> {
```

Comment:
This is called `getUserRole` but it returns a variable named `smallestTask`, I am a bit confused